### DISCUSSION TITLE
Crazy network question

### DISCUSSION THREAD

**Discussion by calebjeffery** [geerlingguy/raspberry-pi-pcie-devices#154] [General]
I am no network guru but just wondering how many of these could you put into a series is daisy chain as a means extending a network?

I do expect laughs from this question but there is no such thing as a dumb question

**Comment by brokensyntax** [comment]
You mean how many different Raspberry Pis could you deploy, ever further away from the primary router as Wireless extenders/Routers?
Many :D
Look at the wireless mesh/wireless darknet projects and you'll get an idea how large such networks can be.

I recall a project out there that uses them to deploy emergency communication networks in natural disasters.
